src/sandbox/sandbox_linux.go

```diff
@@ -65,22 +66,19 @@ func Sandbox(args []string) error {
 			return fmt.Errorf("Failed to bring loopback interface up: %s", err)
 		}
 	}
-
-	return cmd.Run()
+	err = syscall.Exec(cmd, args, env)
+	return fmt.Errorf("Failed to exec %s: %s", cmd, err)
```

Comment:
do you want to always return an error?

Reply:
An error will only be returned if it failed to execute `cmd`. I think `syscall.Exec` uses `execve` under the hood which replaces the current process, so this function would never return on success.

Reply:
Indeed :D